sos/plugins/omnipath_manager.py

```diff
@@ -0,0 +1,50 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+from sos.plugins import Plugin, RedHatPlugin
+import os
+
+
+class OmnipathManager(Plugin, RedHatPlugin):
+    """OmniPath Fabric Manager
+    """
+
+    plugin_name = 'omnipath_manager'
+    profiles = ('hardware',)
+
+    packages = ('opa-fm')
+
+    def setup(self):
+
+        self.add_cmd_output([
+            "/usr/lib/opa-fm/bin/config_check -v -d -s",
```

Comment:
Why are these all hidden away in this location? How is the administrator expected to run them if they are required? Are they added to PATH by some script in `/etc/profile.d`?